script.js

```diff
@@ -1,4 +1,151 @@
+// there are 4 mode .
+// 1) player 1 roll . 2) player 1 pick to make the highest number.
+// 3) player 2 roll. 4) player 2 pick to make the highest number.
+var mode = 'player 1 roll dice';
+
+// store player 1 highest number
+var playerOneHighestNum = 0;
+
+var randomDice1 = 0;
+var randomDice2 = 0;
+
+// There are 2 dice.
+var diceRoll1 = function () {
+  var randomDiceNum = Math.floor(Math.random() * 6) + 1;
+  return randomDiceNum;
+};
+
+var diceRoll2 = function () {
+  var randomDiceNum2 = Math.floor(Math.random() * 6) + 1;
+  return randomDiceNum2;
+};
+
+// player one roll 2 dice logic
+var rollTwoDiceForPlayerOne = function () {
+  var message = '';
+  randomDice1 = 0;
+  randomDice2 = 0;
```

Comment:
Why is it necessary to initialise these variables to 0 when you assign them values again on lines 28-29?